src/third_party/progress/progress.hpp

```diff
@@ -47,7 +47,7 @@ class C_Progress
     /** @brief Initializer of the C_Progress class
      * @param expected_count The number of step of the process
```

Comment:
@bjornpiltz We must update the doxygen doc here since you added the msg variable.